docs-source/content/userguide/introduction/introduction.md

```diff
@@ -16,7 +16,7 @@ Detailed instructions are available [here]({{< relref "/userguide/managing-opera
 
 ### Prerequisites
 
-* Kubernetes 1.11.5+, 1.12.3+, and 1.13.0+  (check with `kubectl version`).
+* Kubernetes 1.11.0+, 1.12.3+, and 1.13.0+  (check with `kubectl version`).
```

Comment:
1.12.0 is not allowed? I thought we were saying 1.11 and higher was all we required.

Reply:
Let's say 1.12.0.

Reply:
Got it.

Reply:
@rjeberhard I don't agree with this change.  We listed what we tested.  We should not change it without doing the necessary testing.